doc/api/tls.markdown

```diff
@@ -183,8 +183,10 @@ automatically set as a listener for the [secureConnection][] event.  The
 
   - `dhparam`: A string or `Buffer` containing Diffie Hellman parameters,
     required for Perfect Forward Secrecy. Use `openssl dhparam` to create it.
-    If omitted or invalid, it is silently discarded and DHE ciphers won't be
-    available.
+    Its key length should be greater than or equal to 1024bits, otherwise
+    it throws an error. It is recommended to use 2048bits or more for
+    enough strong security. If omitted or invalid, it is silently
+    discarded and DHE ciphers won't be available.
```

Comment:
* `s/1024bits/1024 bits/`, ditto for 2048bits.

* I would `s/recommended/strongly recommended/` and maybe add a warning that someday we're going to enforce it.

* `s/enough strong security/stronger security/`